plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -104,8 +104,9 @@
 
     /** First version with new parent POM. */
     private static final String CORE_NEW_PARENT_POM = "1.646";
+    public static final String JENKINS_CORE_FILE_REGEX = "WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:-(?i)(alpha|beta|rc)-[0-9.]+)?(?:-SNAPSHOT)?)[.]jar";
```

Comment:
I would make dash after (alpha|beta|rc) optional in order to support versions like 2.7.3-rc1